README.md

```diff
@@ -1,108 +1,186 @@
-supercomputers, national grids, academic and commercial clouds. Airavata builds
-on general concepts of service oriented computing, distributed messaging, and
-workflow composition and orchestration. Airavata bundles a server package with
-an API, client software development Kits and a general purpose reference UI
-implementation -
-[Apache Airavata Django Portal](https://github.com/apache/airavata-django-portal).
+## 🚀 About
 
-Learn more about Airavata at
-[https://airavata.apache.org](https://airavata.apache.org).
+Apache Airavata is a modern, scalable software framework designed for executing and managing computational jobs across distributed computing resources. Whether you're working with local clusters, supercomputers, national grids, or cloud platforms, Airavata provides a unified interface for computational workflow management.
 
-## Building Apache Airavata
+**Key Features:**
+- 🔧 Service-oriented architecture with distributed messaging
+- 🔄 Workflow composition and orchestration
+- ☁️ Multi-cloud and hybrid cloud support
+- 🖥️ Comprehensive API and SDK ecosystem
+- 🌐 Reference UI implementation via [Apache Airavata Django Portal](https://github.com/apache/airavata-django-portal)
 
-### Prerequisites
+> Learn more at [airavata.apache.org](https://airavata.apache.org)
 
-- Sources compilation requires Java SDK 11.
-- The project is built with Apache Maven 3+.
-- Set or export JAVA_HOME to point to JDK. For example in Ubuntu:
-`export JAVA_HOME=/usr/lib/jvm/adoptopenjdk-11`
-- Git
+## 📋 Prerequisites
 
-### Build the distribution
+Before building Apache Airavata, ensure you have:
 
-    git clone https://github.com/apache/airavata.git
-    cd airavata
-    mvn clean install
+| Requirement | Version | Notes |
+|-------------|---------|-------|
+| **Java SDK** | 17+ | Set `JAVA_HOME` environment variable |
+| **Apache Maven** | 3.8+ | Build automation tool |
+| **Git** | Latest | Version control |
 
-To build without running tests, use `mvn clean install -Dmaven.test.skip=true`.
-The compressed binary distribution is created at
-PROJECT_DIR/modules/distribution/target.
+### 🔧 Environment Setup
 
-### Build and run docker distribution (Experimental and not recommended for production deployments)
-
-* This requires docker and docker-compose installed in your system
+**Ubuntu/Debian:**
+```bash
+export JAVA_HOME=/usr/lib/jvm/openjdk-17-jdk
+```
 
-* Build the source and docker images for each microservice
+**macOS:**
+```bash
+export JAVA_HOME=$(/usr/libexec/java_home -v 17)
 ```
-    git clone https://github.com/apache/airavata.git
-    cd airavata
-    mvn clean install 
-    mvn docker:build -pl modules/distribution
+
+**Windows:**
+```powershell
+set JAVA_HOME=C:\Program Files\OpenJDK\openjdk-17
 ```
-* Start supporting services and Airavata miroservices (API Server, Helix Components and the Job Monitors)
+
+## 🏗️ Building from Source
+
+### Quick Start
+
+```bash
+# Clone the repository
+git clone https://github.com/apache/airavata.git
```

Comment:
Shall we go with the SSH-based version?  (including line 83)

`git clone git@github.com:apache/airavata.git`